src/shadowbox/docker/run_action.sh

```diff
@@ -17,13 +17,17 @@
 do_action shadowbox/docker/build
 
 readonly OUTLINE_DIR=/tmp/outline
-mkdir -p $OUTLINE_DIR && touch "$OUTLINE_DIR/config.json"
+readonly STATE_DIR=$OUTLINE_DIR/persisted-state
```

Comment:
You can call is `HOST_STATE_DIR` to make it super clear. I like that you called the other one `CONTAINER_STATE_DIR`

Reply:
Done